### PR TITLE
Small M3-VL Ballistics Vest Addition To Requisition Vendors

### PR DESCRIPTION
**Pull request by KoishiVibe**
Adds M3-VL vests to Requisitions armor vendors.
# About the pull request

Adds 5 M3-VL vests to the Requisitions vendors at the start of the game. It's in the same format as the other entries so it *should* work...?

# Explain why it's good for the game

Medical and other non-combatant deployed personnel may now be "in uniform" with scrubs _and_ bodyarmor. Helps with easy identification of support personnel... for better or worse. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Added 5x M3-VL Ballistics Vests for issue to noncombat and support personnel deployed to the mission. 
/:cl:

### CODE REVIEW

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 1]
```diff
@@ -392,6 +392,7 @@
 		list("M3 Pattern Skull Marine Armor", 20, /obj/item/clothing/suit/storage/marine/skull, VENDOR_ITEM_REGULAR),
 		list("M3-EOD Pattern Heavy Armor", 10, /obj/item/clothing/suit/storage/marine/heavy, VENDOR_ITEM_REGULAR),
 		list("M3-L Pattern Light Armor", 10, /obj/item/clothing/suit/storage/marine/light, VENDOR_ITEM_REGULAR),
+		list("M3-VL Pattern Ballistics Vest", round(scale*1.5), /obj/item/clothing/suit/storage/marine/light/vest, VENDOR_ITEM_REGULAR),
```
Comment (fira) [inline code/game/machinery/vending/vendor_types/requisitions.dm:395]: That won't work. Scale is only usable in `populate_product_list` and is mainly for balance. If you jsut want a static amount of things in the vendor like above lines, you can just put in the number.

**Review by fira — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending/vendor_types/requisitions.dm (hunk 1 above)
Comment (KoishiVibe) [reply code/game/machinery/vending/vendor_types/requisitions.dm:395]: Oh fuck. Okay. Darn.

**Comment by Steelpoint** [comment]
M3-VL is basically medium armour with light armour speed. It'd be excessively strong in the hands of regular marines. You may want to make a sub-type of this armour that uses light-armour defence values, in my opinion. 

I'm actually quite surprised how strong the armour is, now that I think about it.

**Comment by KoishiVibe** [comment]
> M3-VL is basically medium armour with light armour speed. It'd be excessively strong in the hands of regular marines. You may want to make a sub-type of this armour that uses light-armour defence values, in my opinion. If only cause internal damage.

I wonder, though. M3VL has zero bio armor so you take full damage from acid and slightly more damage from melee given the 5 less. Though I don't know if that trips any breakpoints...

But this is all fair. I mean there are some unused old images of body armor and bulletproof vests. 
...what, open a PR for another M3 variant, for issue to medical personnel, as well as replacing the ones on the ship currently?

**Review by Zonespace27 — APPROVED** [review]
Thought about it, seems fine (except possibly HvH but :shrug:)

**Pull request closed without merging**